e2e-tests/insurance/cypress/e2e/journeys/policy-and-exports/about-goods-or-services/about-goods-or-services.spec.js

```diff
@@ -160,14 +160,14 @@ context('Insurance - Policy and exports - About goods or services page - As an e
     });
 
     describe('after submitting the form', () => {
-      it('should retain the `type of policy and exports` task status as `completed`', () => {
+      it('should retain the `type of policy and exports` task status as `in progress`', () => {
         cy.navigateToUrl(url);
 
         cy.completeAndSubmitAboutGoodsOrServicesForm();
 
         cy.navigateToUrl(`${INSURANCE_ROOT}/${referenceNumber}${ALL_SECTIONS}`);
 
-        const expected = TASKS.STATUS.COMPLETED;
+        const expected = TASKS.STATUS.IN_PROGRESS;
```

Comment:
This doesn't seem valid, "about goods or services" is the last form so the task should be "complete" after this - I think maybe fixed and reverted after WIP changes

Reply:
policyContact comes after it

Reply:
Of course, sorry - Have you moved this test ("task updates to completed") into policy contact E2E tests for when name is the same, and when the name is different?

Reply:
yes I have